test/Microsoft.SqlTools.ServiceLayer.UnitTests/LanguageServer/LanguageServiceTests.cs

```diff
@@ -210,5 +214,45 @@ public void GetDefaultCompletionListWithMatchesTest()
             Assert.AreEqual(1, result.Length);
 
         }
+
+        [Test]
+        public async Task CheckForNonTSqlLanguageTest()
+        {
+            // get the test service 
+            LanguageService service = TestObjects.GetTestLanguageService();
+            var connInfo = new ConnectionInfo(null, null, null);
+
+            const string tsqlText = "SELECT * FROM sys.objects\n--returning\nselect * s\n";
+            const string nonTSqlText = "returning";
+            string errorLimitText = string.Concat(Enumerable.Repeat("select * s\n", 51));
+
+            var scriptFileTSql = new ScriptFile();
```

Comment:
Ideally, we should separate out every test case instead of combining them. Help in debugging when tests are failing.
Something like 4 separate test cases will be nice.

1. TestValidSqlSyntax.
2. TestInvalidSqlSyntax.
3. Test error threshold
4. Test Non Tsql keyword as object name in valid sql